apps/src/code-studio/initApp/loadApp.js

```diff
@@ -435,6 +442,11 @@ function loadAppAsync(appOptions) {
         }
 
         const store = getStore();
+
+        // Note: We aren't guaranteed that currentUser.signInState will have been set at this point.
+        // It gets set on document.ready. However, it is highly unlikely that the server will return
+        // a response before document.ready is triggered. So far, this hasn't caused problems, so not
+        // worrying about this for now.
```

Comment:
Note: This was just something I noticed while exploring options around using what the client knows about the signed-in user. I didn't end up going down that path for this change, but felt it was worth documenting. If it's confusing here, I can split it into a separate PR, but it seemed benign enough to keep.